src/style.css

```
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star-active.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(n+1) {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url("images/star.svg");
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
+
+.stars--5 .stars__star:nth-child(n+6) {
```

you can merge the styles
```suggestion
.stars--0 .stars__star:nth-child(n+1),
.stars--1 .stars__star:nth-child(n+2),
.stars--2 .stars__star:nth-child(n+3),
.stars--3 .stars__star:nth-child(n+4),
.stars--4 .stars__star:nth-child(n+5),
.stars--5 .stars__star:nth-child(n+6) {
```